docs/subnets/reference-cli-commands.md

```diff
@@ -0,0 +1,517 @@
+# Avalanche-CLI Commands
+
+## Subnet
+
+The subnet command suite provides a collection of tools for developing
```

Comment:
I propose reference to be completely based on CLI command help, and probably we can do something to do automatic generation of it

Reply:
It is identical. I'd definitely be down to automate it, not sure how but I suggest we address in a follow up ticket. https://github.com/ava-labs/avalanche-cli/pull/467 will bring us to parity on the CLI side.